### PR TITLE
Fixes a runtime when observing before roundstart

### PR DESCRIPTION
**Pull request by morrowwolf**
# About the pull request

WHAT MODE

HOW DO WE EXIST

# Explain why it's good for the game

Bug bad

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Morrow
fix: Fixed a runtime when observing before roundstart
/:cl:

### CODE REVIEW

**Review by fira — CHANGES_REQUESTED** [review]
Nonononono this is a real error, you can't fix it like this (or i'd already have, i'm lazy), because this means people that observed won't know it's a pred round once the game starts... 

you'd need to also call toggle_predator_action on starting the game

**Comment by morrowwolf** [comment]
My hubris

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -1065,7 +1065,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(!(RoleAuthority.roles_whitelist[key_to_use] & WHITELIST_PREDATOR))
 		return
 
-	if(SSticker.mode.flags_round_type & MODE_PREDATOR)
+	if(SSticker.mode && SSticker.mode.flags_round_type & MODE_PREDATOR)
```
Comment (harryob) [inline code/modules/mob/dead/observer/observer.dm:1068]:
```suggestion
	if(!SSticker.mode)
		SSticker.OnRoundstart(CALLBACK(src, PROC_REF(toggle_predator_action))
		return

	if(SSticker.mode.flags_round_type & MODE_PREDATOR)
```

**Review by harryob — CHANGES_REQUESTED** [review]
😎

File: code/modules/mob/dead/observer/observer.dm [hunk 2]
```diff
@@ -1065,6 +1065,10 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(!(RoleAuthority.roles_whitelist[key_to_use] & WHITELIST_PREDATOR))
 		return
 
+	if(!SSticker.mode)
+		SSticker.OnRoundstart(CALLBACK(src, PROC_REF(toggle_predator_action))
```
Comment (harryob) [inline code/modules/mob/dead/observer/observer.dm:1069]:
```suggestion
		SSticker.OnRoundstart(CALLBACK(src, PROC_REF(toggle_predator_action)))
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**